connectors/src/connectors/notion/lib/connectors_db_helpers.ts

```diff
@@ -218,3 +218,65 @@ export async function getNotionDatabaseFromConnectorsDb(
 
   return NotionDatabase.findOne({ where });
 }
+
+/**
+ * Get children *that are pages* of a given notion page or database
+ *
+ * !! Not children *of a page*
+ * @param dataSourceInfo
+ * @param notionId
+ * @returns
+ */
+export async function getPageChildrenOfDocument(
```

Comment:
nit: I find `Document` to be hard to understand in this context. Now that I read the rest of the code I understand that `Document` means `Page | Database`. Maybe `Resource` would be clearer ? and maybe `notionPageOrDbId` would be clearer than `notionId` ? No strong opinion on the latter, but just pointing out it took me a minute to figure it out